events/wc_offmap_events.txt

```diff
@@ -653,6 +653,7 @@ character_event = {
 	
```

Comment:
Oh shit, here we go again. `events/wc_offmap_events.txt` has a lot of duplicated lines that can be replaced by scripted everything. So far, it's just pain to check it.

Reply:
You mean the new stuff or the things that were already in the file?

Reply:
New stuff.

Reply:
you mean the on_siege sacrifice stuff?

Reply:
No idea. I stopped checking it after I've seen duplicates.